src/models/User.js

```diff
@@ -1,9 +1,45 @@
 export default (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
-    firstName: DataTypes.STRING,
-    lastName: DataTypes.STRING,
-    email: DataTypes.STRING,
-    password: DataTypes.STRING,
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        is: {
+          args: ['^[a-z]+$', 'i'],
+          msg: 'firstname can only contain alphabets'
+        }
+      }
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        is: {
+          args: ['^[a-z]+$', 'i'],
+          msg: 'firstname can only contain alphabets'
+        }
+      }
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
```

Comment:
Please review the PR for Sign Up validations.
https://github.com/andela/rogue-backend/pull/26